x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/selectors.ts

```diff
@@ -4,6 +4,46 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import qs from 'querystring';
+import URL from 'url-parse';
 import { AlertListState } from '../../types';
 
 export const alertListData = (state: AlertListState) => state.alerts;
+
+export const alertListPagination = (state: AlertListState) => {
+  return {
+    pageIndex: state.request_page_index,
+    pageSize: state.request_page_size,
+    resultFromIndex: state.result_from_index,
+    total: state.total,
+  };
+};
+
+export const isOnAlertPage = (state: AlertListState) => {
+  return URL(state.url).pathname.endsWith('/alerts');
+};
+
+export const paginationDataFromUrl = (state: AlertListState) => {
```

Comment:
can you add a return type to this?

Reply:
if this still returns a single value for `?nerds=1&nerds=2` instead of an array, can you replace the url parsing library